.github/workflows/deploy-to-eks.yaml

```diff
@@ -0,0 +1,134 @@
+name: Create EKS cluster, deploy kubeflow-mlflow Terraform solution and run UATs
+on:
+  pull_request:
+  workflow_dispatch:
+    inputs:
+      k8s_version:
+        description: 'Kubernetes version to be used for the EKS cluster. Make sure that the corresponding K8s version is supported by the cloud.'
+        required: false
+      uats_branch:
+        description: 'Branch to run the UATs from e.g. main or track/1.10. By default, this is defined by the dependencies.yaml file.'
+        required: false
+  schedule:
+    - cron: "17 02 * * 1"
+
+env:
+  CLUSTER_NAME: kubeflow-eks
+  AWS_REGION: ${{ secrets.AWS_REGION }}
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+  JUJU_CONTROLLER: kubeflow-controller
+  JUJU_MODEL: kubeflow
+  CKF_CHANNEL: "1.10/stable"
```

Comment:
question: same as the `MLFLOW_CHANNEL` env, where are these envs being used?
my suggestion for the `JUJU_CONTROLLER` and `JUJU_MODEL` is to remove them, there's no need to parameterize those. For the `CKF_CHANNEL` let's decide it in the other discussion.